code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -48,9 +48,10 @@
 	set category = "Malfunction"
 	set name = "Upgrade Turrets"
 	src.verbs -= /mob/living/silicon/ai/proc/upgrade_turrets
-	for(var/obj/machinery/turret/turret in machines)
-		turret.health += 30
-		turret.shot_delay = 20
+	for(var/obj/machinery/porta_turret/turret in machines)
+		if(turret.ai) //Make sure only the AI's turrets are affected.

```

Comment:
turret.ai is a setting for whether the turret will attack everything (except synths) or fall back to other targeting directives (and can be changed at any time), not if it belongs to an AI or not.